data/remnant/remnant ships.txt

```diff
@@ -942,15 +927,16 @@ ship "Merganser"
 	outfits
 		"Inhibitor Cannon" 2
 		
-		"Millennium Cell" 2
+		"Millennium Cell"
 		"Crystal Capacitor" 2
 		"Small Heat Shunt"
 		"Quantum Key Stone"
 		
-		"Crucible-Class Thruster"
 		"Anvil-Class Engine"
+		"Crucible-Class Thruster"
+		"Crucible-Class Steering"
 		"Bellows-Class Afterburner"
-		"Scram Drive"
```

Comment:
Remnant ships have reverse thrust, so a scram drive isn't as necessary. (Though bad turning on top of this may be an issue)

Reply:
The Merganser is far from tanky, even in its un-nerfed configuration. It's a fast interceptor. 

Two inhibitor cannons with room for one more is your definition of heavily armed?

It's an interceptor. It's built for speed. But that doesn't mean it should be useless at everything else. Your changes were too extreme.

Reply:
> 8000 total HP in a 30 ton frame is extremely tanky.

As I mentioned before, the mass of the frame is meaningless other than in calculations of acceleration and turning rate (and heat dissipation, I guess, but that's less relevant). It's probably so low so that it does not become unwieldy after it's loaded with outfits.

It has less total HP than a T1.2 Lightning Bug and _substantially_ less than the Cool Breeze you keep comparing it to (which isn't a completely implemented ship) and any of the Heliarch, Bunrodea, Ka'het, or Korath LWs. In fact, you have to dig into Human or Gegno tech to find a LW with more total HP, and even then, it's close.